risc0/zkvm/src/host/recursion/tests.rs

```diff
@@ -14,28 +14,44 @@
 
 use std::collections::VecDeque;
 
+use anyhow::Result;
 use risc0_binfmt::tagged_struct;
-use risc0_circuit_recursion::CircuitImpl;
+use risc0_circuit_recursion::{
+    prove::{poseidon254_hal_pair, poseidon2_hal_pair},
+    CircuitImpl,
+};
 use risc0_zkp::{
     adapter::{CircuitInfo, PROOF_SYSTEM_INFO},
     core::digest::{digest, Digest, DIGEST_SHORTS},
     field::baby_bear::BabyBearElem,
 };
-use risc0_zkvm_methods::{multi_test::MultiTestSpec, MULTI_TEST_ELF, MULTI_TEST_ID};
-use test_log::test;
+use risc0_zkvm_methods::{
+    multi_test::MultiTestSpec, MULTI_TEST_ELF, MULTI_TEST_ID, MULTI_TEST_V2_USER_ID,
+};
+use rstest::*;
+use rstest_reuse::*;
```

Comment:
`use *` makes it a little bothersome to figure out where a symbol is coming from when reading the source code.
```suggestion
use rstest::rstest;
use rstest_reuse::{apply, template};
```

Reply:
This is test code, not sure we need to be that interested in it being explicit?